north/src/runtime/console.rs

```diff
@@ -201,9 +229,25 @@ async fn connection(stream: TcpStream, mut tx_main: EventTx) {
 
         // Deserialize message
         let message: Message = serde_json::from_slice(&buffer)?;
+        let tmp_installation_dir = tempdir()?;
+        tx.send(match message.payload {
+            Payload::Installation(size, name) => {
+                debug!("Was an installation message for {} ({} bytes)", name, size);
```

Comment:
Debug log is confusing. change to eg. `Incoming installation request for {} with size {}`.